bears/js/JSONFormatBear.py

```diff
@@ -7,17 +7,12 @@
 from coalib.results.Diff import Diff
 from coalib.results.Result import Result
 from coala_utils.param_convertion import negate
+from coalib.misc.Compatability import JSONDecodeError
 
```

Comment:
Will do it, I read that in mozilla's newcomers guide, maybe we add the same in ours too :D